readthedocs/search/tasks.py

```diff
@@ -210,5 +210,5 @@ def record_search_query(project_slug, version_slug, query, total_results, time_s
         version=version,
         query=query,
     )
-    obj.created = time
+    obj.modified = time
```

Comment:
This line should be removed.
`modified` field gets updated automatically.

Also, this can be simplified to
```python
SearchQuery.objects.create(
    project=project,
    version=version,
    query=query,
)
```